tests/func/conftest.py

```diff
@@ -61,6 +61,20 @@ def tables(self, *a, **kw):
             *a, **kw
         )
 
+    def config(self, role):
+        """ Get a dictionary of configuration settings for a role. """
+        result = self.scalar(
+            "SELECT rolconfig FROM pg_roles WHERE rolname='{rolname}'".format(
```

Comment:
You may prefer to delegate more work to PostgreSQL :

``` sql
SELECT unnest(rolconfig) FROM pg_roles ...
```

Allows you to loop parameters as rows, and you won't have null values. Use `select1` to loop rows of a single value.

Reply:
Good point - done